.github/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Awesome Books 2023</title>
+</head>
+<body>
+	<h1>My Book Collection</h1>
+	<form id="add-book-form">
+		<label for="title-input">Title:</label>
+		<input type="text" id="title-input" required>
+		<label for="author-input">Author:</label>
+		<input type="text" id="author-input" required>
+		<button type="submit">Add</button>
```

Comment:
<img width="742" alt="1" src="https://user-images.githubusercontent.com/77843050/219621139-d63cd732-b4dd-4b55-bc23-39d2800c23fa.png">

<img width="407" alt="3" src="https://user-images.githubusercontent.com/77843050/219621160-b6240a8c-ff46-4c8b-9271-1810becd74bc.png">

- [ ] Hey @Mugiira-Kim, I reaaly appreciate your code syntax, is is easy to understand, keep the great work dear ❤️ , but I remarked some missing requirements as shown in the image above, it us asked to have a `remove` button in order to remove the item from the collection, of corse I was able to remove the book too but you have added the `eventListener` on the book itself which is not what required, please fix this issue, I know you are smater to do it in a few 😆